pynest/nest/tests/test_refractory.py

```diff
@@ -189,7 +195,7 @@ def test_refractory_time(self):
             vm_params = {"interval": resolution, "record_from": [name_Vm]}
             vm = nest.Create("voltmeter", params=vm_params)
             sd = nest.Create("spike_detector", params={'precise_times': True})
-            cg = nest.Create("dc_generator", params={"amplitude": 900.})
+            cg = nest.Create("dc_generator", params={"amplitude": 1200.})
```

Comment:
This is needed to make the clopath model spike?

Reply:
Correct.